### ISSUE TITLE
Show version number

### ISSUE THREAD

**Issue by walter-1** [microsoft/etl2pcapng#62]
running **etl2pcapng.exe /?** or just **etl2pcapng.exe** or any invocation for converting an Etl should tell the current version number `1.10.0`
```
C:\>etl2pcapng.exe
etl2pcapng <infile> <outfile>
Converts a packet capture from etl to pcapng format.
```

Reason: user should be informed, if his local version is up-to-date (compared to info given on https://github.com/microsoft/etl2pcapng/releases) 
- other tools like SysInternals tools always show the current version :-)

**Comment by maolson-msft** [comment]
I agree!

**Comment by maolson-msft** [comment]
Addressed by #73
